schemas/common/descriptors/relationshipdescriptor.schema.json

```diff
@@ -40,7 +40,7 @@
   },
   "allOf": [
     {
-      "$ref": "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/schemadescriptor#/definitions/descriptor"
```

Comment:
This is being considered as remove action in allOf, because of which build is failing with incompatibility changes.